docs/reference/environment-variables.mdx

```diff
@@ -227,6 +227,11 @@ You can find these credentials on the
 [Algolia Dashboard](https://www.algolia.com/dashboard/api-keys), on the **API
 keys** page from the menu.
 
+### Attraqt
+
+`FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY`: your attraqt search api key, found in
+your XO console under `Search > API Keys`
```

Comment:
what about the directly link here too ?

Reply:
Same as above, those are not generic URLs

Reply:
I added a link to the console though as Damien suggested below.